include/roboteam_ai/stp/invariants/game_states/FreeKickThemGameStateInvariant.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by jordi on 28-04-20.
+//
+
+#ifndef RTT_FREEKICKTHEMGAMESTATEINVARIANT_H
+#define RTT_FREEKICKTHEMGAMESTATEINVARIANT_H
+
+#include "stp/invariants/BaseInvariant.h"
+
+namespace rtt::ai::stp::invariant {
+
+/**
+ * Invariant for the game state free kick them prepare
+ */
+class FreeKickThemGameStateInvariant : public BaseInvariant {
```

Comment:
Uninformative class documentation. I could also have said the same by looking at the class name. So please go a little bit more into depth what it really does and why it is useful...